src/bundle/Resources/public/js/scripts/core/collapse.js

```diff
@@ -1,11 +1,58 @@
-(function (global, doc) {
+(function (global, doc, bootstrap) {
+    let toggleAllTimeout;
+    const toggleAllBtn = doc.querySelector('.multi-collapse-btn');
+    const expandAll = toggleAllBtn?.querySelector('.ibexa-attribute-group__toggler-expand');
+    const collapseAll = toggleAllBtn?.querySelector('.ibexa-attribute-group__toggler-collapse');
+    const MULTI_COLLAPSE_BODY_CLASS = '.multi-collapse';
+    let singleElementClicked = [];
+    const checkIfChangeText = () => {
+        const allGroups = doc.querySelectorAll(MULTI_COLLAPSE_BODY_CLASS);
+        if (singleElementClicked.length === allGroups.length || singleElementClicked.length === 0) {
+            collapseAll.classList.toggle('d-none');
```

Comment:
I my opinion we should avoid using BS class `d-none`. I prefer toggle btn label not class for two labels node. But I you want toggle class do it on btn then you won't have to change the class for two nodes, just one, and you can manage showing labels in CSS

Reply:
I followed the existing examples of implementation in the code, but I can change it in any other way.